packages/react-hooks/src/useLenientThresholdPercentage.ts

```diff
@@ -0,0 +1,37 @@
+// Copyright 2017-2023 @polkadot/react-hooks authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import type { Perquintill } from '@polkadot/types/interfaces/runtime';
+
+import getCommitteeManagement from '@polkadot/react-api/getCommitteeManagement';
+import { BN, BN_QUINTILL } from '@polkadot/util';
+
+import { createNamedHook } from './createNamedHook.js';
+import { useApi } from './useApi.js';
+import { useCall } from './useCall.js';
+
+const BACKWARD_COMPATIBLE_LENIENT_THRESHOLD = 90;
+
+// preserved percentage precision is four digits - e.g. 57.75%
+const PERCENTAGE_MUL = new BN(10_000);
+
+function useLenientThresholdPercentageImpl (): number | undefined {
+  const { api } = useApi();
+
+  const lenientThresholdFn = getCommitteeManagement(api).query.lenientThreshold;
+  const isLenientThresholdDefined = !!lenientThresholdFn;
```

Comment:
Seems to me that lack of `lenientThresholdFn` doesn't just say that the lenient threshold is not defined, but more precisely that it's not even fetchable? If so, I'd change it, cause it being undefined can happen for more reasons than just `lenientThresholdFn` missing (e.g. pending fetching).
```suggestion
  const isLenientThresholdFetchable /* or some better name 😅 */ = !!lenientThresholdFn;
```

Reply:
I meant `defined` as in `this variable is defined since aleph-node version vX.Y.Z` not as `is not equal to undefined`. It's perhaps an unfortunate name, because js have `undefined` primitive :sweat_smile:

if you think it's confusing perhaps `isLenientThresholdDefinedOnChain` will do?

Reply:
Something along these lines would help imho; or `doesChainDefineLenientThreshold`. Either one.